yj10_driver:
  serial_device: "/dev/ttyUSB0"
  fake_connect: false

  initial_joint_position:
    joint1: -1.985884347884066e-05
    joint2: -0.795430485397165
    joint3: -1.0591188945439547
    joint4: 0.19748105760656087
    joint5: -1.2843279256917992e-05
